docs/Modules/multichain.mdx

```diff
@@ -34,49 +165,117 @@ yarn add @biconomy/modules
+    rpcUrl: // Base Goerli Rpc
     entryPointAddress: DEFAULT_ENTRYPOINT_ADDRESS,
-    defaultValidationModule: multiChainModule,
+    defaultValidationModule: multiChainModule, // this can also be ecdsaModule instance (MUST if SA is already created with ecdsa as default module)
     activeValidationModule: multiChainModule
   });
 
 let polygonAccount = await BiconomySmartAccountV2.create({
     chainId: ChainId.POLYGON_MUMBAI,
     paymaster: //polygonPaymaster,
     bundler: //polygonBundler,
+    rpcUrl: // Polygon Mumbai Rpc
     entryPointAddress: DEFAULT_ENTRYPOINT_ADDRESS,
-    defaultValidationModule: multiChainModule,
+    defaultValidationModule: multiChainModule, // this can also be ecdsaModule instance
     activeValidationModule: multiChainModule
   });
 
 ```
+
+### Integration - Signing and Sending Multichain userops🌟
+
+```typescript
+
+
+  // This could be swap action, mint nft, transfer ownership, enable different module as such (check use cases above) which you intend to do on multiple chains
+  // You can also batch with providing just array of transactions
+  // If the smart account is not deployed it would be deployed as part of this transaction. No additional action needed in below payload 
+  const transaction = {
+    to: <DESTINATION_CONTRACT_ADDRESS>, 
+    data: <DESTINATION_CONTRACT_METHOD_ENCODED_DATA>,
+  };
+
+  // Build partial userOp for chain1
+  let partialUserOp1 = await baseAccount.buildUserOp([transaction], { 
+    // Assuming Sponsorship Paymaster is to be used otherwise leave as it's optional
+    paymasterServiceData: {
+    mode: PaymasterMode.SPONSORED,
+  }});
+
+  // Build partial userOp for chain2
+  let partialUserOp2 = await polygonAccount.buildUserOp([transaction], { 
+    // Assuming Sponsorship Paymaster is to be used otherwise leave as it's optional
+    paymasterServiceData: {
+    mode: PaymasterMode.SPONSORED,
+  }});
+
+  // Use multichain module to sign once for all ops
+  const returnedOps = await multiChainModule.signUserOps([{userOp: partialUserOp1, chainId: ChainId.BASE_GOERLI_TESTNET}, {userOp: partialUserOp2, chainId: ChainId.POLYGON_MUMBAI}]);
+
+
+  const userOpResponse1 = await baseAccount.sendSignedUserOp(returnedOps[0] as any);
+  // You can also just wait for transaction hash by waitForTxHash()
+  const transactionDetails1 = await userOpResponse1.wait();
+
+  const userOpResponse2 = await polygonAccount.sendSignedUserOp(returnedOps[1] as any);
+  const transactionDetails2 = await userOpResponse2.wait();
+```
+
+:::info
+If Smart Account is not deployed yet then it makes sense to pass multiChainModule as defaultValidationModule. That was it would get enabled by default and has all Ecdsa Module (popular go-to module) capabilities. 
+:::
+
+
+
+
+### Conclusion
+
+It reduces user friction for dApps which for example want to quickly and seamlessly deploy and configure Smart Accounts on several chains or issue session keys with the permissions that vary from chain to chain.
```

Comment:
can we simplify this?

Reply:
It makes it easier for dApps to deploy and set up Smart Accounts across multiple chains or issue session keys with different permissions for each chain, reducing user difficulties.

Reply:
done